docs/source/kedro_project_setup/configuration.md

```diff
@@ -185,8 +185,8 @@ parameters = conf_loader.get("parameters*", "parameters*/**")
 
 This will load configuration files from any subdirectories in `conf` that have a filename starting with `parameters` or are located inside a folder with name starting with `parameters`.
 
-```eval_rst
-.. note::  Since ``local`` is set as the environment, the configuration path ``conf/local`` takes precedence in the example above. Hence any overlapping top-level keys from ``conf/base`` will be overwritten by the ones from ``conf/local``.
+```{note}
+Since ``local`` is set as the environment, the configuration path ``conf/local`` takes precedence in the example above. Hence any overlapping top-level keys from ``conf/base`` will be overwritten by the ones from ``conf/local``.
```

Comment:
```suggestion
Since `local` is set as the environment, the configuration path `conf/local` takes precedence in the example above. Hence any overlapping top-level keys from `conf/base` will be overwritten by the ones from `conf/local`.
```